packages/crayons-icon/utils/svgo.runner.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-undef */
+const fs_extra = require('fs-extra');
+const path = require('path');
+const svgo = require('svgo');
+const log = require('./log');
+
+module.exports = build;
+async function build(rootDir, srcDir, distDir, loadConfigPlugins) {
+  try {
+    let alert = {};
+    const srcSvgDir = path.join(rootDir, srcDir);
+    const distSvgDir = path.join(rootDir, distDir);
+
+    alert = {
+      type: 'info',
+      name: 'Optimizing SVGs In-Progress...',
+      msg: `Target Path: ${srcSvgDir}`,
+    };
+    log(alert);
+
+    await fs_extra.emptyDir(distSvgDir);
+    const srcSvgData = await getSvgs(srcSvgDir, distSvgDir);
+    await optimizeSvgs(srcSvgData, loadConfigPlugins);
+
+    alert = {
+      type: 'success',
+      name: 'DONE',
+      msg: `Optimized SVGs moved to ${distSvgDir}`,
+    };
+    log(alert);
+  } catch (e) {
+    console.error(e);
+    process.exit(1);
+  }
+}
+async function optimizeSvgs(srcSvgData, loadConfigPlugins) {
+  let pluginOptions = [];
+  loadConfigPlugins.plugins.forEach(function (data) {
+    for (key in data) {
+      if (data[key]) pluginOptions = [...pluginOptions, data];
+    }
+  });
+  const optimizePass = new svgo({
+    plugins: [...pluginOptions],
+  });
+
+  await Promise.all(
+    srcSvgData.map(async (svgData) => {
+      return optimizeSvg(optimizePass, svgData);
+    })
+  );
+}
+async function optimizeSvg(optimizePass, svgData) {
+  const srcSvgContent = await fs_extra.readFile(svgData.srcFilePath, 'utf8');
+  const optimizedSvg = await optimizePass.optimize(srcSvgContent, {
```

Comment:
Centralized level in build block, Tested. I cannot add try-catch{} in all functions as it will result in eslint error: 'Unnecessary try-catch wrapper '. So done at the central level.(fn)build.
Fixed
![Uploading Screenshot 2021-12-20 at 3.44.08 PM.png…]()

Reply:
<img width="805" alt="Screenshot 2021-12-20 at 3 44 08 PM" src="https://user-images.githubusercontent.com/76547218/146753648-1e7972a7-56a7-44ad-8f16-758d31cef114.png">

Reply:
<img width="1175" alt="Screenshot 2021-12-20 at 4 05 32 PM" src="https://user-images.githubusercontent.com/76547218/146753824-f08a33a2-ea52-4edc-8f02-da228781ca2f.png">